### ISSUE TITLE
Uncaught error in client creation

### ISSUE THREAD

**Issue by szymon-rooam** [braintree/braintree-web#606]
### General information

* SDK version: 3.83.0
* Environment: Sandbox and production
* Browser and OS: Chrome 96.0.4664 Windows, Mobile Safari 15.0, IOS, 

### Issue description

When connection to braintree gateway fails (Both graphql and standard one) promise is in status rejected 
client.js, line 126   
 `analytics.sendEvent(promise, 'custom.client.load.initialized');`
fails with unhandled rejection 

Steps to replicate: 
run client create script and in  dev-tools block traffic to
 `braintree-api.com/graphql` and `api.sandbox.braintreegateway.com/merchants/`

stack trace:
```  
sendAnalyticsEvent @client.js?f06b:2677
Client.initialize  @client.js?f06b:807
create  @client.js?f06b:1418
eval  @client.js?f06b:314
```

**Comment by crookedneighbor** [comment]
Sounds to me like something on your end is blocking requests to Braintree domains. Do you have a content security policy or a browser extension that may be doing this?

If not, please reach out to our support team with a staging site so we can troubleshoot it: https://developer.paypal.com/braintree/help/

**Comment by szymon-rooam** [comment]
Main thing is that this `analytics.sendEvent(promise, 'custom.client.load.initialized');` is being called even in case of failure to connect to Braintree apis which causes unhandled error inside that i cannot process with rest of my code even if I can handle main rejection from client

**Comment by crookedneighbor** [comment]
This should be fixed in v3.84.0.
